app/com/arpnetworking/metrics/portal/reports/impl/EmailSender.java

```diff
@@ -32,12 +50,84 @@
 public class EmailSender implements Sender {
     @Override
     public CompletionStage<Void> send(
+            final Report report,
             final Recipient recipient,
-            final ImmutableMap<ReportFormat, RenderedReport> formatsToSend
+            final ImmutableMap<ReportFormat, RenderedReport> formatsToSend,
+            final Instant scheduled
     ) {
-        throw new RuntimeException();
+        return CompletableFuture.supplyAsync(() -> {
+            try {
+                sendSync(report, recipient, formatsToSend, scheduled);
+            } catch (final IOException e) {
+                throw new CompletionException(e);
+            }
+            return null;
+        });
     }
 
-    EmailSender() {}
+    private String sendSync(
+            final Report report,
+            final Recipient recipient,
+            final ImmutableMap<ReportFormat, RenderedReport> formatsToSend,
+            final Instant scheduled
+    ) throws IOException {
+
+        final String subject = getSubject(report, scheduled);
+        EmailPopulatingBuilder builder = EmailBuilder.startingBlank()
+                .from("no-reply+amp-reporting@dropbox.com")
+                .to(recipient.getAddress())
+                .withSubject(subject);
+
+        for (final RenderedReport rendered : formatsToSend.values()) {
+            builder = addFormat(builder, rendered);
+        }
+
+        LOGGER.info()
+                .setMessage("sending email")
+                .addData("report", report)
+                .addData("recipient", recipient)
+                .log();
+
+        _mailer.sendMail(builder.buildEmail());
+        return "";
+    }
+
+    private String getSubject(final Report report, final Instant scheduled) {
+        final String formattedTime = ZonedDateTime.ofInstant(scheduled, ZoneId.of("UTC")).toString();
+        return "[Report] " + report.getName() + " for " + formattedTime;
```

Comment:
I think this should also be configurable, but that's a bit more complicated than the above since we'd probably have to have it as a format string so I'd be fine on punting on it for now.

Reply:
Good point, and agreed re: punting! I don't see any good string-templating libraries already in the project (though I could easily have missed something), so I've left a TODO and I'm gonna call this resolved.